app/Integrations/GitHub/GitHubPlugin.php

```
@@ -0,0 +1,423 @@
+            case 'PushEvent':
+                return $this->convertPushEvent($externalData, $integration);
+            case 'PullRequestEvent':
+                return $this->convertPullRequestEvent($externalData, $integration);
+            case 'IssuesEvent':
+                return $this->convertIssueEvent($externalData, $integration);
+            default:
+                return [];
+        }
+    }
+    
+    protected function convertPushEvent(array $data, Integration $integration): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$integration'. (Unused Code Rules)

(UnusedFormalParameter)